src/community/oseo/oseo-stac/src/test/java/org/geoserver/ogcapi/stac/LoadJSONTest.java

```diff
@@ -0,0 +1,47 @@
+package org.geoserver.ogcapi.stac;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.File;
+import org.geoserver.config.GeoServerDataDirectory;
+import org.geoserver.data.test.SystemTestData;
+import org.jsoup.nodes.Document;
+import org.junit.Test;
+
+/**
+ * This test class is created separately due to copying collection.ftl file to the data directory in
+ * the tests. If it is not done in another class, it fails the tests that uses collection.ftl
+ */
+public class LoadJSONTest extends STACTestSupport {
+
+    @Override
+    protected void onSetUp(SystemTestData testData) throws Exception {
+        super.onSetUp(testData);
+
+        copyTemplate("/collections-SENTINEL2.json");
+        copyTemplate("/collection.ftl");
+        copyTemplate("/collections.ftl");
+
+        GeoServerDataDirectory dd =
+                (GeoServerDataDirectory) applicationContext.getBean("dataDirectory");
+
+        File file = dd.getResourceLoader().createFile("/readAndEval.json");
+        dd.getResourceLoader().copyFromClassPath("/readAndEval.json", file, getClass());
+    }
+
+    @Test
+    public void testReadAndEvalJSON() throws Exception {
+        Document dom = getAsJSoup("ogc/stac/collections/SENTINEL2?f=html");
+
+        assertEquals("attribute1 => 1 attribute2 => 2", dom.select("h2").text());
+    }
+
+    @Test
+    public void testReadFileOutsideOfDataDir() throws Exception {
```

Comment:
Seems to me that this method doesn't test what it claims to test. Could you clarify, fix it?